Component/Badge/badge.html

```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"

```

Comment:
Add meta descriptions to make it more personalise like.
<meta name="author" content="your name" />
    <meta name="description"
        content="description of library." />